lib/collector/remote-method.js

```diff
@@ -88,7 +90,32 @@ RemoteMethod.prototype.invoke = function invoke(payload, nrHeaders, callback) {
       if (err) {
         return callback(err)
       }
-      this._post(serialized, nrHeaders, callback)
+      this._post(
+        serialized,
+        nrHeaders,
+        function onResponse(err, res) {
+          if (err) {
+            return callback(err)
+          }
+          if (this._metrics) {
+            const payloadByteLength = byteLength(serialized)
```

Comment:
it might be good to extract out this bit of code into its own function so it's clearer the intention when first reading it